app/src/main/java/com/android/shelfLife/ui/overview/IndividualFoodItem.kt

```diff
@@ -63,6 +64,14 @@ fun IndividualFoodItemScreen(
                   }
             })
       },
+      // Floating Action Button to edit the food item
+      floatingActionButton = {
+        FloatingActionButton(
+            onClick = { navigationActions.navigateTo(Screen.EDIT_FOOD) },
+            content = { Icon(Icons.Default.Edit, contentDescription = "Edit") },
+            modifier = Modifier.testTag("editFoodFab"),
```

Comment:
## Missing test
Unfortunately, there is no other mention of this test tag (editFoodFab) in our codebase. Either remove the tag, or implement a UI test to avoid confusion on what is tested or not.
```suggestion
           
```